cmd/server/cadence/cadence.go

```diff
@@ -145,3 +158,32 @@ func BuildCLI() *cli.App {
 	return app
 
 }
+
+func startTestWebhookEndpoint() {
+	http.HandleFunc("/", logIncomingRequest)
+
+	fmt.Printf("Starting server for testing...\n")
+	if err := http.ListenAndServe(":8080", nil); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func logIncomingRequest(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
+	case "POST":
+		var body []byte
+		body, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			panic(err)
```

Comment:
Maybe log errors instead of panic -- this will exit the whole process(including notifier)